vavr/src/main/java/module-info.java

```diff
@@ -0,0 +1,7 @@
+module io.vavr {
```

Comment:
Is this still Java 8 compatible?
@danieldietrich do we need a separate, modularized Java 9 release also?